spmd/tensor/parallel/utils.py

```diff
@@ -43,17 +43,15 @@ def _prepare_input_validate(
     @functools.wraps(_prepare_input_func)
     def wrapper(*args, **kwargs):  # pyre-ignore[2, 3]
         assert len(args) >= 1, "_prepare_input needs at least one arg."
-        # args is an immutable tuple but we may need to modify it
-        args = list(args)
+        device_mesh = None
```

Comment:
How about `device_mesh = None if len(args) < 2 else args[1]`?

Reply:
If still have lint errors I will change it, lol.

Reply:
Changed. Thanks!